src/runtime/threading_backend.cc

```diff
@@ -148,19 +146,36 @@ class ThreadGroup::Impl {
           CPU_SET(sorted_order_[0], &cpuset);
         }
         pthread_setaffinity_np(pthread_self(), sizeof(cpu_set_t), &cpuset);
+      } else {
```

Comment:
Under current logic, if you set `TVM_BIND_MASTER_THREAD=1`, the issue fixed in #4344 will come back. I would suggest keeping the current way, but making sure binding it to the right big/small core that user sets.

Reply:
Hmm...How about call SetFullCpuAffinity(reverse) directly like Android? I think make master thread could have free migration on needed cores is better than bind just one specific core. Thanks.

Reply:
Sounds good. And I am fine with enabling the master thread to migrate across cores.

Reply:
Good. I will fix it when I back to work ASAP.